Bottom: d51a790af9f15a8f2a95c58ab9cfa5fdde3dbff6
Top:    085f7c15662b316080106da090760de1b5abe8c5
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-08-22 12:33:54 -0700

Refresh of cmtime-ret

---

diff --git a/examples/cmtime.c b/examples/cmtime.c
index 220e675..6761f2c 100644
--- a/examples/cmtime.c
+++ b/examples/cmtime.c
@@ -208,9 +208,8 @@ static void cma_handler(struct rdma_cm_id *id, struct rdma_cm_event *event)
 		break;
 	case RDMA_CM_EVENT_ADDR_ERROR:
 		if (n->retries--) {
-			ret = rdma_resolve_addr(n->id, rai->ai_src_addr,
-						rai->ai_dst_addr, timeout);
-			if (!ret)
+			if (!rdma_resolve_addr(n->id, rai->ai_src_addr,
+					       rai->ai_dst_addr, timeout))
 				break;
 		}
 		printf("RDMA_CM_EVENT_ADDR_ERROR, error: %d\n", event->status);
@@ -219,8 +218,7 @@ static void cma_handler(struct rdma_cm_id *id, struct rdma_cm_event *event)
 		break;
 	case RDMA_CM_EVENT_ROUTE_ERROR:
 		if (n->retries--) {
-			ret = rdma_resolve_route(n->id, timeout);
-			if (!ret)
+			if (!rdma_resolve_route(n->id, timeout))
 				break;
 		}
 		printf("RDMA_CM_EVENT_ROUTE_ERROR, error: %d\n", event->status);
@@ -363,7 +361,7 @@ static int run_client(void)
 
 	conn_param.responder_resources = 1;
 	conn_param.initiator_depth = 1;
-	conn_param.retry_count = 5;
+	conn_param.retry_count = retries;
 	conn_param.private_data = rai->ai_connect;
 	conn_param.private_data_len = rai->ai_connect_len;
